library/components/TFiltersDropdown.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+  <t-dropdown :is-open="isOpen" @open="isOpen = true" @close="isOpen = false">
+    <!-- Handle button -->
+    <div slot="handle" class="flex items-center gap-1 p-1 text-sm font-medium">
+      <plus-icon :size="18" />
+      Add filter
+      <menu-down-icon :size="18" />
+    </div>
+
+    <!-- Search input -->
+    <div class="p-2 filters-search roboto-fonts">
+      <div class="search-input">
+        <magnify-icon :size="18" />
+        <input
+          class="bg-transparent outline-none !text-black"
+          placeholder="Filters by..."
+          v-model="search"
+        />
+        <div class="w-5">
+          <close-icon v-if="search" :size="18" @click="search = ''" />
+        </div>
+      </div>
+    </div>
+
+    <div class="flex flex-col gap-2 px-2 pb-2 tracking-widest roboto-fonts">
+      <div v-if="!filteredItems.length">No filters found</div>
+      <div v-for="(section, index) in filteredItems" :key="section">
+        <div class="px-2 py-1 text-xs font-semibold">{{ section.label }}</div>
+        <div class="text-sm">
+          <div
+            class="px-2 py-1 rounded hover:bg-[#F9F8FA]"
+            v-for="item in expanded.includes(index)
+              ? section.items
+              : section.items.slice(0, itemLimit)"
+            :key="item"
+            @click="toggleFilter(item)"
+          >
+            {{ item.label }}
+          </div>
+        </div>
+        <div
+          @click="handleExpanded(index)"
+          class="text-[#145DEB] text-sm px-2 py-1"
+          v-if="section.items.length > 4"
+        >
+          <span v-if="expanded.includes(index)"> Show less </span>
+          <span v-else>
+            Show all {{ section.label.toLowerCase() }} filters
+          </span>
+        </div>
+      </div>
+    </div>
+  </t-dropdown>
+</template>
+
+<script>
+export default {
```

Comment:
Could you please add:
`name: "TFiltersDropdown",`
? It helps the vue devtools.

Reply:
Good idea @msorber updated.